sdk/servicefabricmanagedclusters/Microsoft.Azure.Management.ServiceFabricManagedClusters/tests/TestHelper/RecordedDelegatingHandler.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace ServiceFabricManagedClusters.Tests.TestHelper
+{
+
+    using System;
+    using System.Net;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Threading.Tasks;
+
+    public class RecordedDelegatingHandler : DelegatingHandler
```

Comment:
Why are you adding this?  There is already a similar delegating handler in the test framework

Reply:
removed